app/src/pages/settings/SettingsAnnotationsPage.tsx

```diff
@@ -152,7 +160,16 @@ const SettingsAnnotations = ({
     <Card
       title="Annotation Configs"
       variant="compact"
-      bodyStyle={{ padding: 0 }}
+      bodyStyle={{
+        padding: 0,
+        // Subtract the height of the card header. Without this, the last row of the table is cut off.
+        // TODO: Remove this once we refactor the card to use the new design system
+        // we can solve with height variables, or with flex grow throughout the card so that the card children are aware of their own height
+        maxHeight: "calc(100% - 47px)",
+        flex: "1 1 auto",
+        display: "flex",
+        flexDirection: "column",
+      }}
```

Comment:
In general I've started to avoid card bodies that scroll and just have content scroll. I always feel "trapped" in these scrolling areas and if there's no gutter on both sides it's even worse. Is it possible to just fix this on the entire config section as overflow scroll rather than this specific card?

Reply:
That's possible yeah, we just lose sticky table headers if so

Reply:
I think that's my preference - leaves room for other cards to be in the mix. The sticky header only seems helpful when you are trying to deal with very large scrolling tables.

Reply:
Closing this in favor of https://github.com/Arize-ai/phoenix/pull/7253